docker/lr-utils/Makefile

```diff
@@ -1,11 +1,15 @@
+IMAGE_NAME = lr-utils
 VERSION = 0.1.10
```

Comment:
given all the changes brought in by the changes in Dockerfile, this deserves a change.

Reply:
👍